dorianim/Orion
# Update build.gradle.kts

### Motivation and Context



### Description


### Steps for Testing


1.  Install the release as described in [the readme](https://github.com/ls1intum/Orion/blob/master/README.md#testing-of-pull-requests)
2.  Check if all features still work (start and clone exercise, submit changes, receive test results, clone exercise as instructor)
3.  ...

### Screenshots
